napari/settings/_tests/test_settings.py

```diff
@@ -302,10 +302,12 @@ def test_get_settings(tmp_path):
 def test_get_settings_fails(monkeypatch, tmp_path):
     p = f'{tmp_path}.yaml'
     settings.get_settings(p)
-    with pytest.raises(Exception) as e:
+    with pytest.raises(
+        RuntimeError, match='The path can only be set once per session'
+    ):
         settings.get_settings(p)
 
-    assert 'The path can only be set once per session' in str(e)
+    # assert 'The path can only be set once per session' in str(e)
```

Comment:
I need to uncomment this. I was having trouble with this locally. Didnt mean to push this.